packaging/test_irods_client_rest_cpp.py

```diff
@@ -275,6 +275,26 @@ def test_list(self):
             finally:
                 shutil.rmtree(dir_name)
                 admin.assert_icommand(['irm', '-f', '-r', dir_name])
+    def test_list_honors_recursive__issue114():
+        with session.make_session_for_existing_admin() as admin:
+            try:
+                pwd, _ lib.
```

Comment:
i don't know this syntax... can you explain/document/make more plain?

Reply:
This is ancient programming wisdom, unsuitable for everyday use...